test/other/test_null_deref_via_js.c

```diff
@@ -0,0 +1,7 @@
+#include <emscripten.h>
+
+int main() {
+  EM_ASM({
+    HEAP32[0] = 0;
```

Comment:
How about `return HEAP32[0]`  since that would be more like a null deref?

Also, do we not have tests for this already elsewhere?

Reply:
> How about `return HEAP32[0]` since that would be more like a null deref?

Well one is load, another is store, but both result in null deref. Either is fine by me.

> Also, do we not have tests for this already elsewhere?

Apparently not because nothing caught this before.

Reply:
lgtm as is then

Reply:
Ah found test_safe_heap_user_js.c that I can reuse.